### PR TITLE
feat: implement new design for built-in style

### PR DESCRIPTION
**Pull request by hanny21**
# Description

# Checklist

- [ ] `make test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows [commit guidelines](./doc/guides/commit-message.md#commit-message-guidelines)

### CODE REVIEW

File: lib/services/core/ui/components/buttons.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ const navButtonUI = ({ label, href, id, onClick }) => {
     aEl.href = href;
   }
   aEl.innerText = label;
-  aEl.style.cssText = "color: rgb(255, 255, 255); text-decoration: none;";
+  aEl.style.cssText = `color: rgb(250, 250, 250); text-decoration: none;`;
```
Comment (dusan-kovacik-profiq) [inline lib/services/core/ui/components/buttons.js:50]: I thought the linter doesn't allow us to use backticks when there's no text substitution.

File: lib/services/core/__integration_tests__/renderLogin.test.js [hunk 2]
```diff
@@ -180,11 +178,6 @@ describe("when server returns a logical message type", () => {
...
         "button#IKUISDK-btn-oidc-indykite-me.indykite-me-login-button.oidc-button",
       );
-      expect(indykiteMeLoginButton).toBeTruthy();
-      const divider1 = form.nextSibling;
-      expect(divider1.tagName.toLowerCase()).toBe("hr");
-      const divider2 = otherOptionsLabel.previousSibling;
-      expect(divider2.tagName.toLowerCase()).toBe("hr");
     });
 
     describe("when the 'Create an account' link is clicked", () => {
```
Comment (lorigoldman) [inline lib/services/core/__integration_tests__/renderLogin.test.js:183]: Does case matter? If it does, pls. use "When the ..."

File: README.md [hunk 3]
```diff
@@ -192,7 +192,6 @@ const Registration = () => {
         confirmPassword: "Custom Confirm Password",
         registerButton: "Custom Join",
         alreadyHaveAnAccountButton: "Custom Already have an account",
```
[the remark this answers is not in the document]
Comment (hanny21) [reply README.md:194]: will fix it, thanks

File: lib/services/core/exports/IKUICore/renderLogin.js [hunk 4]
```diff
@@ -11,7 +11,6 @@ const throttleChecker = require("./throttleChecker");
  * @param {string} [props.labels.loginButton]
  * @param {string} [props.labels.registerButton]
  * @param {string} [props.labels.forgotPasswordButton]
- * @param {string} [props.labels.orOtherOptions]
  * @param {{ [optionId: string]: string; }} [props.loginApp]
  * @param {string} [props.redirectUri] In case you use OIDC providers you need to specify the callback redirect Uri
```
Comment (lorigoldman) [inline lib/services/core/exports/IKUICore/renderLogin.js:15]: providers, ...URI.

File: lib/services/core/exports/IKUICore/renderLogin.js [hunk 5]
```diff
@@ -11,7 +11,6 @@ const throttleChecker = require("./throttleChecker");
...
  * @param {string} [props.registrationPath="/registration"] Url where user will be redirect after clicking on register button.
```
Comment (lorigoldman) [inline lib/services/core/exports/IKUICore/renderLogin.js:16]: URL?

File: lib/services/core/exports/IKUICore/renderRegister.js [hunk 6]
```diff
@@ -11,7 +11,6 @@ const render = require("./render");
  * @param {string} [props.labels.confirmPassword]
  * @param {string} [props.labels.registerButton]
  * @param {string} [props.labels.alreadyHaveAnAccountButton]
- * @param {string} [props.labels.orOtherOptions]
  * @param {{ [optionId: string]: string; }} [props.loginApp]
  * @param {string} [props.redirectUri] In case you are using OIDC providers with option to register.
  * @param {string} [props.loginPath="/login"] Url where user will be redirect after clicking on login button.
```
Comment (lorigoldman) [inline lib/services/core/exports/IKUICore/renderRegister.js:16]: redirected

File: lib/services/core/locale/src/en.json [hunk 7]
```diff
@@ -8,18 +8,18 @@
   "uisdk.general.error": "Oops something went wrong. Please try again.",
```
Comment (lorigoldman) [inline lib/services/core/locale/src/en.json:8]: Oops,

File: lib/services/core/locale/src/en.json [hunk 8]
```diff
@@ -8,18 +8,18 @@
...
-  "uisdk.general.other_options": "Or continue with",
   "uisdk.login.login": "Login",
   "uisdk.login.register_button": "Create an account",
   "uisdk.login.forgot_password_button": "Forgot password",
+  "uisdk.login.oidc_text": "Log in with",
   "authapi.err.validate_email_verdict": "username policy requires valid email",
   "authapi.err.password_too_short": "{authapi.err.password_too_short, plural, one {password too short, must be more than %d character} other {password too short, must be more than %d characters}}",
   "authapi.err.password_too_long": "{authapi.err.password_too_long, plural, one {password too long, must be less than %d character} other {password too long, must be less than %d characters}}",
   "authapi.err.username_password_policy_validation": "username or password policy validation failed",
   "authapi.err.no_recovery_methods": "you have not set up a recovery methods for your account. contact system administrator",
```
Comment (lorigoldman) [inline lib/services/core/locale/src/en.json:19]: a recovery method for your account. Contact your system administrator.

**Review by lorigoldman — APPROVED** [review]

**Review by maprokes — CHANGES_REQUESTED** [review]
@hanny21 could you update the commit message to follow the conventional commit for breaking changes?

> BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with [MAJOR](http://semver.org/#summary) in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.

https://www.conventionalcommits.org/en/v1.0.0/

**Review by dusan-kovacik-profiq — APPROVED** [review]
I think after you fix the comments here it's good to go ;)

**Pull request merged**